api/api.yaml

```diff
@@ -153,19 +158,36 @@ definitions:
       es_field_type:
         type: string
         description: The Elasticsearch field type.
-      values:
+      value_names:
```

Comment:
Delete 166-167

Put on 161:

If this is a regular facet (https://i.imgur.com/ug1mSEr.png), only value_names and value_counts are set.
If this is a time series facet (https://i.imgur.com/IU2SqWv.png), then value_names, time_names, time_series_value_counts are set. Consider https://i.imgur.com/IU2SqWv.png: the data is a table with 6 rows and 3 columns. Column names are time_names; row names are value_names; time_series_value_counts are the numbers in the table.